languages.yaml

```
@@ -1427,6 +1427,37 @@ Lua:
       - '#808080'
       - '#FFFFFF'
     chip: '#000080'
+Makefile:
+  type: programming
+  ascii: |
+    {0}    _-`````-,           ,- '- .
+    {0}  .'   .- - |          | - -.  `.
+    {0} /.'  /                     `.   \
+    {0}:/   :      {1}_...   ..._      {0}``   :
+    {0}::   :     {1}/._ .`:'_.._\.    {0}||   :
+    {0}::    `._ {1}./  ,`  :    \{0} . _.''   .
+    {0}`:.      {1}/   |  -.  \-. \\_{0}      /
+    {0}  \:._ {1}_/  .'   .{2}@{1})  \{2}@{1}) ` `\{0} ,.'
+    {1}    {0}'{1}_/,--'       .- .\,-.`--`.
+    {1}       ,'/''     (( \ `  )    
+    {1}        /'/'  \    `-'  (      
+    {1}         '/''  `._,-----'
+    {1}          ''/'    .,---'
+    {1}           ''/'      ;:
+    {1}             ''/''  ''/
+    {1}               ''/''/''
+    {1}                 '/'/'
+    {1}                  `;
+  colors:
+    ansi:
+      - white
+      - yellow
+      - red
+    hex:
+      - '#FFFFFF'
+      - '#FAEC9A'
+      - '#610000'
+    chip: '#610000'
```

```suggestion
    chip: '#427819'
```

cf. linguist entry for [Makefile](https://github.com/github/linguist/blob/c1c34e5260797b4d598f5ec76f19723bfc5a1894/lib/linguist/languages.yml#L3833)

---

Commited https://github.com/o2sh/onefetch/pull/921/commits/5bf867ffb59b9e662c627ff46019a6bf4e15eaeb thanks for the suggestion!